packages/components/collapsible-panel/src/collapsible-panel.tsx

```diff
@@ -241,13 +275,21 @@ const CollapsiblePanel = (props: TCollapsiblePanel) => {
               )}
             </HeaderContainer>
             <div style={containerStyles}>
-              <SectionWrapper ref={registerContentNode}>
+              <SectionWrapper
+                // @ts-ignore
```

Comment:
I tried several things but I don't find how to make the typing work for this reference.

Any help would be much appreciated 🙏

Reply:
I think I used type assertion in `<RichTextBody>` for forwarded refs so I'd suggest:
```suggestion
import type { ReactNode, Ref } from 'react';

<SectionWrapper
  ref={registerContentNode as Ref<HTMLDivElement>}
```